scripts/submit_csv.sh

```diff
@@ -28,12 +28,32 @@ shift
 TARGET=${1:-2}
 shift
 
+# environment variable to indicate whether the job is running on condor or slurm
+SYSTEM=${SYSTEM:-condor}
+
 # process csv file into jobs
-CSV_FILE=$($(dirname $0)/csv_to_chunks.sh ${FILE} ${TARGET})
+if [ -n "${CSV_FILE:-}" ]; then
+  # allow to set custom csv file for job instead of fetching from web archive
+  CSV_FILE=$(realpath -e ${CSV_FILE})                     
+else
+  CSV_FILE=$($(dirname $0)/csv_to_chunks.sh ${FILE} ${TARGET})
+fi
 
 # create command line
-EXECUTABLE="./scripts/run.sh"
-ARGUMENTS="${TYPE} EVGEN/\$(file).\$(ext) \$(nevents) \$(ichunk)"
+EXECUTABLE="$PWD/scripts/run.sh"
+if [ ${SYSTEM} = "condor" ]; then
+  ARGUMENTS="${TYPE} EVGEN/\$(file).\$(ext) \$(nevents) \$(ichunk)"
+elif [ ${SYSTEM} = "slurm" ]; then
+  ARGUMENTS="${TYPE}"
+  # FIXME: This is not ideal. It prevents from submitting multiple jobs with different JUG_XL_TAG simultaneously.
+  cd scripts 
```

Comment:
See above for `SCRIPTDIR` which would be useful here.

Also pushd-popd for directory stacks then.